docs/how-to-guides/develop-a-tool/how-to-use-enabled-by.md

```diff
@@ -0,0 +1,218 @@
+# How to use enabled by in the tool package
+
+This guide will instruct you on how to use the "enabled by" feature in the tool package. The "enabled by" feature is designed to display which inputs are enabled when a customer uses a specific input type or input value. This feature is particularly useful if you need to adapt your inputs based on varying requirements.
```

Comment:
Cascading settings between inputs are frequently used in situations where the selection in one input field determines what sebsequent inputs should be shown. This approach help in creating a more efficient, user-friendly, and error-free input process. To support cascading settings between tool inputs, we've introduced following two configurations:
**enabled_by + enabled_by_type**: This setting enabled the current input based on the value type of the dependent input.
**enabled_by + enabled_by_value**: This setting enabled the current input based on the actual value of the dependent input.